Tests/HardwareTests/Gpio.test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <Sts1CobcSw/Hal/Gpio.hpp>
+#include <Sts1CobcSw/Hal/PinNames.hpp>
+#include <type_safe/types.hpp>
+
+#include <rodos_no_using_namespace.h>
+
+#include <array>
+
+
+namespace sts1cobcsw
+{
+
+// pa2, pa3 are tested by printing to UCI UART
+auto pinsToTest = std::to_array<RODOS::HAL_GPIO>(
```

Comment:
- missing `constexpr`
- Global variables are bad. Turn this into a static member of `GpioTest`.